### ISSUE TITLE
Accessibility for ChoiceGroup is not aligned with ARIA radio group

### ISSUE THREAD

**Issue by sampascal** [OfficeDev/office-ui-fabric-react#11103]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue: 
Fabric Choicegroup doesn't support accessibility as ARIA intructed. https://www.w3.org/TR/wai-aria-practices/examples/radio/radio-1/radio-1.html

When focus changed from one choice to the other, the selection got changed as well.
The expected behavior described by ARIA is not the same.

### Please provide a reproduction of the issue in a codepen:

This issue can be reproduced directly on Fabric documentation.

**Comment by JasonGore** [comment]
The example you provided changfes selection when focus moves as well:
![RadioGroup](https://user-images.githubusercontent.com/26070760/68347108-ceab6200-00aa-11ea-9baf-f87f160eabb7.gif)

 `ChoiceGroup` appears to be following the [ARIA Radio Group design guidelines](https://www.w3.org/TR/wai-aria-practices/#radiobutton):

> * Right Arrow and Down Arrow: move focus to the next radio button in the group, uncheck the previously focused button, and check the newly focused button. If focus is on the last button, focus moves to the first button.
> * Left Arrow and Up Arrow: move focus to the previous radio button in the group, uncheck the previously focused button, and check the newly focused button. If focus is on the first button, focus moves to the last button.

`ChoiceGroup` seems consistent with the ARIA Radio Group design guidelines, unless I misunderstood the issue description.

**Comment by sampascal** [comment]
I think I saw
